src/backend/commands/vacuum.c

```diff
@@ -2000,30 +1968,25 @@ vac_update_datfrozenxid(void)
 		newMinMulti = cached_dbform->datminmxid;
 
 	if (dirty)
-<<<<<<< HEAD
 	{
 		HeapTuple			tuple;
 		Form_pg_database	tmp_dbform;
+		void			   *inplace_state;
 		/*
 		 * Fetch a copy of the tuple to scribble on from pg_database disk
 		 * heap table instead of system cache
 		 * "SearchSysCacheCopy1(DATABASEOID, ObjectIdGetDatum(MyDatabaseId))".
 		 * Since the cache already flatten toast tuple, so the
 		 * heap_inplace_update will fail with "wrong tuple length".
 		 */
-		tuple = fetch_database_tuple(relation, MyDatabaseId);
+		tuple = fetch_database_tuple(relation, MyDatabaseId, &inplace_state);
```

Comment:
For clarity, I suggest removing the `fetch_database_tuple` function and placing its contents in place of its call. Also, it would be good to update the comment above.